src/platform/bouffalolab/BL602/InetPlatformConfig.h

```diff
@@ -43,5 +43,5 @@
 #endif // INET_CONFIG_NUM_TCP_ENDPOINTS
 
 #ifndef INET_CONFIG_NUM_UDP_ENDPOINTS
-#define INET_CONFIG_NUM_UDP_ENDPOINTS 4
+#define INET_CONFIG_NUM_UDP_ENDPOINTS 6
```

Comment:
Where is this "6" coming from?  If it's meant to be "MAX_FABRICS + 1", then maybe that's what it should be set to?

Reply:
I think it's not meant to be "MAX_FABRICS + 1". when system start "DnssdServer::StartServer()", it will consume one udp endpoint. System itself consumed 4 udp endpoint. when bl602 connected to the ap, we will send dnssd msg, it wiil consumed one udp endpoint. Based on the memory limitations of embedded devices, INET_CONFIG_NUM_UDP_ENDPOINTS was set to 6.